api/nodes/filters.py

```diff
@@ -78,16 +79,16 @@ def build_query_from_field(self, field_name, operation):
 class UserNodesFilterMixin(NodesFilterMixin):
     def build_query_from_field(self, field_name, operation):
         if field_name == 'current_user_permissions':
-            if not operation['value'] in PERMISSIONS:
+            if not operation['value'] in API_CONTRIBUTOR_PERMISSIONS:
```

Comment:
Drive-by nit: Prefer `if x not in y:` to yoda conditions.

I realize this is how it was before, though.